src/Illuminate/Database/Connection.php

```diff
@@ -317,7 +317,7 @@ public function getSchemaBuilder()
     /**
      * Begin a fluent query against a database table.
      *
-     * @param  \Closure|\Illuminate\Database\Query\Builder|\Illuminate\Database\Query\Expression|string  $table
+     * @param  \Closure|\Illuminate\Database\Query\Builder|Illuminate\Contracts\Database\Query\Expression|string  $table
```

Comment:
Why was the `\` removed? There is no definition of `\Illuminate\Database\Illuminate\Contracts\Database\Query\Expression`

Reply:
Seems this was fixed post-merge 👍